parkings/models/parking_terminal.py

```diff
@@ -16,5 +16,11 @@ class Meta:
         verbose_name_plural = _("parking terminals")
         ordering = ('number',)
 
+    @property
+    def zone(self):
+        # Avoid circular imports by importing this function here.
+        from parkings.api.enforcement.check_parking import get_payment_zone
+        return get_payment_zone(self.location)
```

Comment:
I think the get_payment_zone function should be moved to another module which could be just imported here. That could be e.g. `parkings.geo_utils`.